src/components/CodeAndPreview.tsx

```diff
@@ -41,12 +41,7 @@ export function CodeAndPreview({ snippet }: Props) {
     )
   }, [])
 
-  // Initialize with template or snippet's manual edits if available
-  const [manualEditsFileContent, setManualEditsFileContent] = useState(
-    snippet?.manual_edits_json ??
-      JSON.stringify(manualEditsTemplate, null, 2) ??
-      "",
-  )
+  const [manualEditsFileContent, setManualEditsFileContent] = useState("{}")
```

Comment:
you're not consistent with the default value- make it the empty string as it is in other places